common/include/obc_errors.h

```diff
@@ -14,12 +15,16 @@ typedef enum {
     OBC_ERR_CODE_TASK_CREATION_FAILED = 102,
     OBC_ERR_CODE_TIMER_CREATION_FAILED = 103,
     OBC_ERR_CODE_MUTEX_CREATION_FAILED = 104,
-
+    OBC_ERR_CODE_QUEUE_CREATION_FAILED = 105,
+    OBC_ERR_CODE_TIMER_START_FAILED = 106,
+    OBC_ERR_CODE_QUEUE_SEND_FAILED = 107,
+    OBC_ERR_CODE_MUTEX_TAKE_FAILED = 108,
```

Comment:
No need for this one. We already have an error code for it called `OBC_ERR_CODE_MUTEX_TIMEOUT`